.github/workflows/pre-release-tests.yml

```diff
@@ -10,14 +10,18 @@ on:
 
 jobs:
   integration_tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
     name: integration-tests-against-rc
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - name: Set up Python 3.7
+      - name: Set up Python
```

Comment:
My local version has this change in it. I'm  not sure why it wasn't included with when I did push. I will make sure it gets added.

Reply:
Oh wait, I see what you did now. Sorry about that, yes I can include this.

Reply:
Since you are using pipenv I added tox-pipenv to the Pipfile which installed tox as a dependency. If you would like me to also add tox to be more explicit I'm happy to do that.

Reply:
Oh it didn't install tox for me! So I manually added tox to test it. Does it work for you without having tox already installed?

Reply:
It did work for me without having tox installed. The simple solution seems to be to add tox to the Pipfile and then it will work the same for everyone so I'll do that (not sure why it worked differently for the two of us).